contracts/staking/StakingToken.sol

```diff
@@ -51,9 +51,6 @@ contract StakingToken is BaseShareField {
 			// burn
 			totalSupply -= value;
 		}
-
-		_decreaseProductivity(from, value);
-		_increaseProductivity(to, value);
```

Comment:
so now i dont think you need to overwrite _transfer at all

Reply:
but where does that happen now if not here?

Reply:
It's in the simplestaking _transfer override